src/test/java/com/google/api/codegen/testdata/discoveries/cloudmonitoring.v2beta2.json.overrides

```diff
@@ -1,13 +1,7 @@
 {
   "java": {
     "methods": {
-      "cloudmonitoring.metricDescriptors.list": {
-        "requestBodyType": null
-      },
-      "cloudmonitoring.timeseries.list": {
-        "requestBodyType": null
-      },
-      "cloudmonitoring.timeseriesDescriptors.list": {
+      ".*list": {
```

Comment:
While unlikely, it's possible that they add another list method that is actually correct.
Should we do `cloudmonitoring.(metricDescriptors|timeseries|timeseriesDescriptors).list" instead?